data/movies_ontology.json

```diff
@@ -26,7 +26,10 @@
         "actors",
         "directors",
         "year",
-        "title"
+        "title",
+        "time",
+        "companion",
+        "location"
```

Comment:
I believe `slots_annotation` is used to fetch information from the database so it might break in the current version. Do we need to extend this list in this PR?